### ISSUE TITLE
Limit data folder size

### ISSUE THREAD

**Issue by odinuv** [keboola/docker-bundle#99]
Od 1.12 umožňuje docker limitovat velikost kontejneru, takže bychom mohli /data šoupnout do kontejneru a ten by mohla bejt nějaká appka, co bude řešit input/output mapping a vyjmulo by se to úplně ven z docker runneru

https://keboola.slack.com/archives/faillog/p1480226810000007

Je potreba overit, jestli docker velikost kontejneru i **garantuje**, abysme nedali appce limit 100gb a v pulce nezjistili, ze to nema kam davat.

**Comment by ondrejhlavacek** [comment]
No, pokud si to místo dopředu nealokuje, což bude trvat, tak to garantovat nemůže. Možná na to bude potřeba jinej volume driver, kterej vytvoří fixně velikej image?

**Comment by Halama** [comment]
pak by to spousteni a alokaci asi mel hlidat docker runner nejak.

**Comment by ondrejhlavacek** [comment]
No, to by asi šlo, pokud se budou ty image fixne alokovat, tak není problém
se podívat, jestli se vleze nebo ne. Velikost alokovanyho disku by asi měla
bejt uložená v definici image. Nejúspornější by bylo, kdyby si to definoval
kazdej run, ale to asi není jak udělat. Ještě by to mohlo bejt per project?

On Nov 29, 2016 8:57 AM, "Martin Halamíček" <notifications@github.com>
wrote:

pak by to spousteni a alokaci asi mel hlidat docker runner nejak.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/keboola/docker-bundle/issues/99#issuecomment-263501341>,
or mute the thread

**Comment by Halama** [comment]
kdyz to bude pro image tak to staci imho zatim.

**Comment by ondrejhlavacek** [comment]
Probírali jsme s Martinem ještě nějaký možnosti

- pro každej job vytvořit a namountovat vlastní EBS volume s potřebnou velikostí
- použít nějakej virtuální FS na OS levelu
- datový kontejnery

u všeho bude potřeba vyřešit cleanup, když proces skončí/spadne - podobně jak na transformacích. To asi nejjednodušeji půjde u datovýho kontejneru, kterej by mohl bejt pojmenovanej tak, že ho půjde identifikovat z cleanup commandu.
